solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -571,6 +581,15 @@ contract RandomBeacon is Ownable {
         relay.requestEntry(groupId);
 
         callback.setCallbackContract(callbackContract);
+
+        // If current request should trigger group creation and group creation
+        // is not currently in progress we should start the warm up process.
```

Comment:
`trigger group creation and group creation`? 😆 

```suggestion
        // If the current request should trigger group creation we need to lock
        // DKG state (lock sortition pool) to prevent operators from changing its
        // state before relay entry is known. That entry will be used as a group
        // selection seed.
```

Reply:
Done as part of https://github.com/keep-network/keep-core/pull/2691/commits/a29a5ad1f24125d53b37cef20129d6c5977764be